test/integration/shebang/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  target: 'serverless',
+}
```

Comment:
Is this target necessary for the test? If not, lets remove the file and use the default.

Reply:
It was automatically generated by Next, so I left it in.